src/server/pkg/deploy/images/images.go

```diff
@@ -112,7 +112,7 @@ func Import(opts *assets.AssetOpts, in io.Reader) error {
 			for _, err := range loopErr {
 				errStr += err.Error() + "\n"
 			}
-			return fmt.Errorf("errors pushing image %s:%s:\n%s", registryRepo, tag, errStr)
+			return errors.Errorf("errors pushing image %s:%s:\n%s", registryRepo, tag, errStr)
```

Comment:
Yeah, this code is a little weird, I think I'm going to change it to just return the first error, rather than trying to return all of them. That's a more standard go thing to do, and chances are they'll be the same error anyways.

Reply:
(And by returning the first error I mean wrapping it.)

Reply:
Alright, moved this over to returning a wrapped copy of the first error.